### PR TITLE
Garage, Turn out, Model fixes

### LINKED ISSUE #157: Model: Blue line showing in cabin for pilot or co-pilot of MH-60 and MH-60 DAP
# Model Error report 

## **Description:**

- When in either the pilot or co-pilot seat of an MH-60 or MH-60 DAP when looking back into the cabin a blue line can be seen running through roughly the middle of the cabin. This does not occur on other versions or from other seats.

### PR DESCRIPTION
**Pull request by ampersand38**
**When merged this pull request will:**
- Add Texture and Animation customization via Garage
- Add Turn Out for door and window gunners https://www.youtube.com/watch?v=4H-7rK-_Uqg, https://www.youtube.com/watch?v=xO6yVlIOjU0
- Fix SFM center of mass https://www.youtube.com/watch?v=IiNMQ0UCBDc
- Fix viewCargo model errors for cabin doors, pylons, gunner seats
- Close #157, #111, #164

### CODE REVIEW

File: addons/uh60_flir/functions/fnc_setup.sqf [hunk 1]
```diff
@@ -58,6 +58,7 @@ vtx_uh60_flir_isVisibleMap = visibleMap;
 call vtx_uh60_flir_fnc_setIsPipHidden;
 
 if (vtx_uh60_flir_playerIsPilot) then {
+  player action ["TurnIn", vehicle player];
```
Comment (ampersand38) [inline addons/uh60_flir/functions/fnc_setup.sqf:61]: Fix #245

**Comment by ampersand38** [comment]
image
image
image
Shadow LOD bits

**Review by YannikH — APPROVED** [review]
LGTM

**Comment by ampersand38** [comment]
Merged in #272

**Pull request closed without merging**